estela-api/builds/config.py

```diff
@@ -0,0 +1,10 @@
+from builds.default import DefaultBuild
+from builds.gcp import GCPBuild
+
+def Build(type):
```

Comment:
We should add a section in the docs describing this class

Reply:
Done.